imgparse/_version.py

```diff
@@ -1,3 +1,3 @@
 """Defines package version.  Parsed by setup.py and imported by __init__.py."""
 
-__version__ = "1.15.3"
+__version__ = "1.15.4"
```

Comment:
Minor version bump for the new functionality

Reply:
Done https://github.com/SenteraLLC/py-image-metadata-parser/pull/70/commits/5f918bc6a026511259eca02bd3d884fa2aec8fef